spec/player_spec.rb

```diff
@@ -1,72 +1,97 @@
 require "./lib/player.rb"
 
 describe Scrabble::Player do
-  before :each do
-    @player = Scrabble::Player.new("Ada")
-  end
+ before :each do
+   @player = Scrabble::Player.new("Ada")
+ end
 
-  describe "#initialize" do
-  it "has a name" do
-    expect(@player.name).to eq("Ada")
-    end
-  end
+ describe "#initialize" do
+ it "has a name" do
+   expect(@player.name).to eq("Ada")
+   end
+ end
 
-  describe "plays" do
-    it "return the words array" do
-    expect(@player.plays).to eq([])
-    end
-  end
+ describe "plays" do
+   it "return the words array" do
+   expect(@player.plays).to eq([])
+   end
+ end
 
-  describe "play" do
-    it "Adds the input word to the plays Array" do
-      @player.play("apple")
-      expect(@player.plays).to eq(["apple"])
-    end
-    it "return an empty array when there are no words" do
-      @player.play("")
-      expect(@player.plays).to eq ([""])
-    end
-    it "Returns false if player has already won" do
-      @player.score = 101
-      expect(@player.play("apple")).to eq(false)
-    end
-  end
+ describe "play" do
+   it "Adds the input word to the plays Array" do
+     @player.play("apple")
+     expect(@player.plays).to eq(["apple"])
+   end
+   it "return an empty array when there are no words" do
```

Comment:
This test's definition in English doesn't match what the test is doing.

The `expect` on line 27 is checking that you get an empty string in the array returned from `plays` after playing that empty string.